lib/cylc/gui/combo_logviewer.py

```diff
@@ -66,7 +66,8 @@ def create_gui_panel(self):
         for snum in snums:
             combobox2.append_text(str(snum))
         combobox2.connect("changed", self.switch_snum)
-        combobox2.set_active(snums.index(self.nsubmit))
+        if snums:
```

Comment:
Probably safest to do `if self.nsubmit in snums`. The `list.index` method can also raise `ValueError` if value is not in the list.

Reply:
Okay, thanks. I'll get this changed accordingly.